infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -82,6 +82,17 @@ void prepare() throws InfrastructureException {
         .kubernetesRoleBindings()
         .inNamespace(namespace)
         .createOrReplace(createViewRoleBinding());
+
+    // If the user specified an additional cluster role for the workspace,
+    // create a role binding for it too
+    String customClusterRoleName = System.getenv("CHE_WORKSPACE_CLUSTER_ROLE");
```

Comment:
it would be better to do the same as with service account name. I mean use configuration property instead of env var directly.
See https://github.com/eclipse/che/blob/7b6cd44e36e94d40f366b5dcd2fe46074e8a7bad/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java#L40

Reply:
P.S. `che.infra.kubernetes.cluster_service_account_name` would be right configuration property name, I guess. Also, add the default value to `che.properties` file

Reply:
@sleshchenko Quick question: If both the environment variable and the Che property are set, which should be used? Or should I remove the environment variable and only use the Che property?